src/labels/CustomLabels.labels

```diff
@@ -1983,6 +1983,22 @@
         <shortDescription>Recurring Donation Entry Form Donor Information section header</shortDescription>
         <value>Donor Information</value>
     </labels>
+    <labels>
+        <fullName>RD2_EntryFormDonorTypeHelpText</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>The Inline HelpText for the Donor Type picklist</shortDescription>
+        <value>Select the type of donor for this recurring donation. A value of &quot;Account&quot; requires entry of an Organization Account and optionally a related Contact. A value of &quot;Contact&quot; will require entry of a Household Contact.</value>
```

Comment:
@sam-knox Would you mind reviewing the help text for the "Donor Type" field on the RD New/Edit UI?

Reply:
@sam-knox Just checking back on this new label.

Reply:
Yes, sorry I missed this. Here's my suggested update to the Value:

"Select the type of donor for this Recurring Donation. If you select Account, enter an Organization Account, and optionally a related Contact. If you select Contact, enter a Contact related to a Household Account."

Reply:
Thanks @sam-knox. Change made.